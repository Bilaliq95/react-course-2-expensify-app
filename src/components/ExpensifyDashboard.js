import React from 'react';
import ExpenseList from './ExpenseList'
import ExpenseListFilter from './ExpenseListFilter'
import getExpensesTotal from '../Selectors/expenses-total'
import expenses from '../tests/fixtures/expenses'
import ExpensesSummary from '../components/ExpensesSummary'
const ExpensifyDashboard=()=>(
    <div>
    <ExpensesSummary/>
    <ExpenseListFilter/>    
     <ExpenseList/>
    </div>
    
    );

 export default ExpensifyDashboard;   