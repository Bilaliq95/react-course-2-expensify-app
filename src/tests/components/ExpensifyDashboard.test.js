import React from 'react'
import ExpensifyDashboard from '../../components/ExpensifyDashboard';
import {shallow} from 'enzyme'

test('should render expensify dashboard component correctly',()=>{
    const wrapper=shallow(<ExpensifyDashboard />)
    expect(wrapper).toMatchSnapshot();
});