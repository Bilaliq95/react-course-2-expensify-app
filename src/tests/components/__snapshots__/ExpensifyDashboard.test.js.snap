// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expensify dashboard component correctly 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
